app/views/layouts/frontend/mailer.html.erb

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
```

Comment:
No need to split layout for email, email have a same layout but different content, subject...